SetupDataPkg/Include/Library/ConfigDataLib.h

```diff
@@ -119,6 +122,26 @@ typedef struct {
   UINT8     BaseTableBitMask[0];
 } ARRAY_CFG_HDR;
 
+#pragma pack(push, 1)
+typedef struct {
+  /* Size of Name in bytes */
+  UINT32    NameSize;
+
+  /* Size of Data in bytes */
+  UINT32    DataSize;
+
+  /*
+   * Rest of Variable List struct:
+   * 
+   * char Name[NameSize] // Null terminate UTF-16LE encoded name

```

Comment:
nit: Probably want to make the data types to be consistently UEFI style.

Reply:
Fixed